src/lib/minject/InjectorMacro.hx

```diff
@@ -177,15 +177,12 @@ class InjectorMacro
 			if (keep.exists(field.name))
 				field.meta.add(':keep', [], Context.currentPos());
 
-		// sort rtti to ensure post constructors are last and in order
-		infos.sort(function (a, b) return a.order - b.order);
-
 		// add rtti to type
-		var rtti = infos.map(function (info) return macro $v{info.rtti});
+		var rtti = infos.map(function (rtti) return macro $v{rtti});
 		if (rtti.length > 0) ref.meta.add('rtti', rtti, ref.pos);
 	}
 
-	static function processField(field:ClassField, infos:Array<{order:Int, rtti:Array<String>}>, keep:Map<String, Bool>):Void
+	static function processField(field:ClassField, rttis:Array<Array<String>>, keep:Map<String, Bool>):Void
```

Comment:
Now that the order is included in the method RTTI, we can just use the RTTI array directly, no need for the anonymous object.